ChangeLog

```diff
@@ -39,6 +39,16 @@ Release date: TBA
   Closes pylint-dev/pylint#10112
 
 
+What's New in astroid 3.3.7?
```

Comment:
We're duplicating this entry in 3.3.8. It happened with the last release as well.